src/components/cms/SkillFeedbackCard/SkillFeedbackCard.js

```diff
@@ -233,6 +259,20 @@ class SkillFeedbackCard extends Component {
                       : `${userEmail.slice(0, userEmail.indexOf('@') + 1)}...`}
                   </div>
                   <Timestamp>{formatDate(parseDate(data.timestamp))}</Timestamp>
+                  {data.rating > 0 ? (
+                    <Ratings
+                      rating={data.rating}
```

Comment:
Move it to a common place, it is repeated at multiple places

Reply:
Made a separate `Rating` component for that.